File: AdminBackend/src/main/java/Backend/AdminBackend/model/ekstenzije/SubjectKeyIdentifierEkstenzije.java
package Backend.AdminBackend.model.ekstenzije;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class SubjectKeyIdentifierEkstenzije {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column
    private boolean daLiKoristi;

    @Column
    private boolean isCritical;

    public SubjectKeyIdentifierEkstenzije(boolean daLiKoristi, boolean isCritical) {
        this.daLiKoristi = daLiKoristi;
        this.isCritical = isCritical;
    }
}

File: AdminFrontend/src/app/MODEL/EKSTENZIJE/SubjectKeyIdentifierEkstenzije.ts
export class SubjectKeyIdentifierEkstenzije {
    daLiKoristi: boolean

    isCritical: boolean
    constructor(daLiKoristi: boolean, isCritical: boolean) {
        this.daLiKoristi = daLiKoristi
        this.isCritical = isCritical
    }
}